login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx

```diff
@@ -115,10 +115,19 @@ export const RegistrationWorkflow: React.FC<React.PropsWithChildren<Registration
     };
 
     const finishRegistration = (data: IndividualScreenData): Promise<void> => {
-        if (actions && actions().completeRegistration)
-            // TODO: THIS LOOKS BROKEN — ARE WE ONLY PASSING THE DATA FROM THE LAST SCREEN OF THE WORKFLOW???
```

Comment:
Can you check with @daileytj, why he added this comment?

Reply:
I believe that came from @joebochill  during the docs cleanup story

Reply:
This PR is to address that todo comment only.